src/librustc_session/parse.rs

```diff
@@ -60,6 +60,17 @@ impl GatedSpans {
     }
 }
 
+#[derive(Default)]
+pub struct SymbolGallery {
+    pub symbols: Lock<FxHashSet<Symbol>>,
+}
+
+impl SymbolGallery {
+    pub fn collect(&self, symbol: Symbol) {
```

Comment:
```suggestion
    pub fn insert(&self, symbol: Symbol) {
```
?
`collect` is usually used in other context.

Reply:
ok~

Reply:
Done